ursula_cli/shell.py

```diff
@@ -195,7 +195,8 @@ def _run_heat(args, hot):
     CREDS = {
         'username': os.environ['OS_USERNAME'],
         'password': os.environ['OS_PASSWORD'],
-        'tenant_name': os.environ['OS_TENANT_NAME'],
+        'tenant_name': (os.environ.get('OS_TENANT_NAME') or
+                        os.environ['OS_PROJECT_NAME']),
```

Comment:
Because the get method will return a false if the key doesn't exist, which allows us to use the `or` to directly grab the fallback key. If neither key exist, an error will bubble up. It's a shortcut around having an if block.